### PR TITLE
Metrics: PSS for a transfer rate of 1Gb

### PR DESCRIPTION
**Pull request by GabyCT**
Measure of the proportional set size memory while doing
a transfer rate of 1Gb using nuttcp.

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

**Comment by chavafg** [comment]
qa-passed

File: Makefile.am [hunk 1]
```diff
@@ -68,6 +68,7 @@ GENERATED_FILES = \
 	tests/metrics/network/network-metrics-cpu-consumption.sh \
 	tests/metrics/network/network-metrics-memory-pss.sh \
 	tests/metrics/network/network-metrics-memory-rss-1g.sh \
+	tests/metrics/network/network-metrics-memory-1g.sh \
```
Comment (grahamwhaley) [inline Makefile.am]: For consistency with the other two related tests, this really should be called `network-metrics-memory-pss-1g.sh` :-)
Could you fix that here and below (and obviously rename the source file itself) please.

**Comment by grahamwhaley** [comment]
Minor naming nit - would be nice if we can fix that.
Other than that, thanks for the rework and rebase - test runs fine here, so with the expectation of the renames:
lgtm

**Comment by GabyCT** [comment]
@grahamwhaley changes of the name applied

**Comment by chavafg** [comment]
qa-passed

File: Makefile.am [hunk 2]
```diff
@@ -372,6 +373,7 @@ EXTRA_DIST = \
 	tests/metrics/network/network-metrics-cpu-consumption.sh.in \
 	tests/metrics/network/network-metrics-memory-rss-1g.sh.in \
 	tests/metrics/network/network-metrics-memory-pss.sh.in \
+	tests/metrics/network/network-metrics-memory-pss-1g.sh.in \
```
Comment (jodh-intel) [inline Makefile.am]: This new test needs to be added to https://github.com/01org/cc-oci-runtime/blob/master/tests/metrics/network/README.md.

Comment (GabyCT) [inline Makefile.am]: @jodh-intel when https://github.com/01org/cc-oci-runtime/pull/946 is merged then I can add it

Comment (GabyCT) [inline Makefile.am]: or I can add another PR for the modification of the README

**Comment by grahamwhaley** [comment]
I think adding the README mods to this PR as another commit is the best thing - sure, you can wait until #946 is merged (although you might not get a conflict...). I'd hope that will get merged pretty quick, so should not block you.

**Comment by GabyCT** [comment]
@jodh-intel changes were applied

**Comment by jodh-intel** [comment]
lgtm

**Comment by jodh-intel** [comment]
@iphutch - please could you review the doc changes?

**Comment by chavafg** [comment]
qa-passed

**Comment by jodh-intel** [comment]
Hi @rcaballeromx, not sure if @iphutch is about so could you review this?

**Comment by jodh-intel** [comment]
Thanks @iphutch!

**Pull request merged**